src/scenario-registry.ts

```diff
@@ -163,6 +164,17 @@ export const scenarios = [
     displayOrder: 1,
     rootComponent: ImagePreview,
   },
+
+  /**
+   * Billing related scenarios
+   */
+  {
+    groupName: 'Billing',
+    shortId: 'subscribe-to-subscription-plan',
+    displayName: 'Subscribe to a Subscription Plan (with Paypal purchase)',
```

Comment:
we could consider shortening this a bit - i.e. `Subscribe to a Subscription Plan (with PayPal)`